src/test/resources/mockdata/quickMarcWrong008Items.json

```diff
@@ -0,0 +1,117 @@
+{
+  "id" : "2f390fa6-a2f8-4027-abaf-ee61952668bc",
+  "leader" : "01542ccm a2200361   4500",
+  "fields" : [ {
+    "tag" : "001",
+    "content" : "393893",
+    "indicators" : [ ]
+  }, {
+    "tag" : "005",
+    "content" : "20141107001016.0",
+    "indicators" : [ ]
+  }, {
+    "tag" : "008",
+    "content" : "{\n  \"Content\" : \"Scores\",\n  \"Type\" : \"c\",\n  \"BLvl\" : \"m\",\n  \"Entered\" : \"830419\",\n  \"DtSt\" : \"m\",\n  \"Date1\" : \"19551\",\n  \"Date2\" : \"99999\",\n  \"Ctry\" : \"gw \",\n  \"Comp\" : \"mu\",\n  \"FMus\" : \"a\",\n  \"Part\" : \" \",\n  \"Audn\" : \" \",\n  \"Form\" : \" \",\n  \"AccM\" : [ \"h\", \"i\", \"z\", \" \", \" \", \" \" ],\n  \"LTxt\" : [ \"n\", \" \" ],\n  \"TrAr\" : \" \",\n  \"Lang\" : \"lat\",\n  \"MRec\" : \" \",\n  \"Srce\" : \" \"\n}",
```

Comment:
Am I correct that this should be object but not a string? cc @NikitaSedyx

Reply:
yes, object